src/Core/src/Platform/Android/EntryExtensions.cs

```diff
@@ -50,6 +50,14 @@ internal static void SetInputType(this EditText editText, IEntry entry)
 
 			if (entry.IsPassword && ((editText.InputType & InputTypes.ClassNumber) == InputTypes.ClassNumber))
 				editText.InputType |= InputTypes.NumberVariationPassword;
+
+			if (!entry.IsTextPredictionEnabled && ((editText.InputType & InputTypes.TextFlagNoSuggestions) != InputTypes.TextFlagNoSuggestions))
+				editText.InputType |= InputTypes.TextFlagNoSuggestions;
```

Comment:
I am thinking this is unnecessary setting of the native property. Even though it is fast and, maybe we can use a variable and pipe them all. Then at the end, we set it?

Reply:
This enhancement is addressed in #507.